### ISSUE TITLE
ReadMe typo

### ISSUE THREAD

**Issue by RafLaf** [hushell/pmf_cvpr22#30]
Hi,
I think I found a typo in the meta-test command line you give. I am not sure though

```python -m torch.distributed.launch --nproc_per_node=8 --use_env test_meta_dataset.py --data-path /path/to/meta_dataset/ --dataset meta_dataset --arch dino_small_patch16 --deploy finetune --output outputs/your_experiment_name --resume outputs/your_experiment_name/best.pth --dist-eval --ada_steps 100 --ada_lr 0.0001 --aug_prob 0.9 --aug_types color transition```

This gives me this error: 
```

pmf_cvpr22/models/utils.py", line 18, in DiffAugment
    for f in AUGMENT_FNS[p]:
KeyError: 'transition' 

```

I think you meant translation instead of transition right?

Cheers,
Raphael

**Comment by RafLaf** [comment]
Actually it's issue #26 already but it hasn't been fixed

**Comment by hushell** [comment]
Fixed by 622d656afba5aeb39b70eaa3cceda74ad0b09faf
